client/src/components/Input.js

```diff
@@ -22,7 +22,9 @@ const Input = ({ onSubmit }) => {
 
 			if (response.ok) {
 				const data = await response.json();
-				onSubmit(data);
+				const correctedSentence = data.choices[0].message.content;
```

Comment:
I suggest you do this: `data.choices[0].message.content` on the server side, so the `data` variable returned from the server is just the string you are interested in